MapzenSDK/MZMapViewController.swift

```diff
@@ -910,6 +910,9 @@ open class MZMapViewController: UIViewController, LocationManagerDelegate {
 
   override open func viewDidLoad() {
     super.viewDidLoad()
+    let httpHandler = TGHttpHandler()
+    httpHandler.httpAdditionalHeaders = MapzenManager.sharedManager.httpHeaders() as! NSMutableDictionary
```

Comment:
This crashes because swift dictionaries aren't bridged to mutable dictionaries. You need something like `httpHandler.httpAdditionalHeaders = NSMutableDictionary(dictionary: MapzenManager.sharedManager.httpHeaders())`

Reply:
Oh? Doesn't crash for me. Hmmmm. Will recheck that.

Reply:
I've retested this inside, outside, and backwards and I cannot get it to crash. Stepping over it shows values getting set correctly on all objects involved. 

There is bridging for Dictionary types documented here: https://developer.apple.com/library/content/documentation/Swift/Conceptual/BuildingCocoaApps/WorkingWithCocoaDataTypes.html

However it's not performing 100% the way I would expect it to based on reading that documentation; Tangram objective-c code imported into Swift should have NSMutableDictionary types replaced with straight swift Dictionary types. However that might only happen reliably if the objective-c code in use is making use of lightweight typing.

That said, I will rework this a bit to avoid the forced downcast using your suggestion which does work.

Reply:
Thats odd, its very reproducible without the fix. Running the sample app results in a crash every time:

<img width="1256" alt="screenshot" src="https://user-images.githubusercontent.com/494323/31516116-9852259e-af54-11e7-9bc6-6687e2d9ad8f.png">
 
Anyway, code looks good now. After a rebase its good to go.